### ISSUE TITLE
Support aliases

### ISSUE THREAD

**Issue by ymhr** [CyCraft/vue-intellisense#8]
I have a vue project using webpack aliases, but the tool seems to error when trying to find files imported using them.

I have `@` set up as an alias to `src/`, but I get errors like this:
```
Neither '@/components/profile/Avatar.vue' nor '@/components/profile/Avatar.js(x)' or '@/components/profile/Avatar/index.js(x)' or '@/components/profile/Avatar/index.ts(x)' could be found in '...[username].../src/components/profile'
```

Would it be possible for this to follow webpack aliases?

**Comment by mesqueeb** [comment]
@ymhr 
I have added the feature to allow for aliases. Currently however it's only possible via the script.
Can you try creating a script like so and try it?

```js
import { generateVeturFiles } from '@vue-intellisense/scripts'

const inputPath = 'string'
const outputPath =  'string'
const options = { recursive: true, alias: { '@': 'src' } }

generateVeturFiles(
  inputPath,
  outputPath,
  options
)
```

and pass the aliases as object.

or if that doesn't work try `{ '@/': 'src/' }`

try that and let me know!

**Comment by mesqueeb** [comment]
closing because of inactivity

**Comment by mesqueeb** [comment]
@ymhr proper alias support released in 0.3.0. thanks to @boydaihungst 

please check readme or cli manual!
